### PR TITLE
feat(github/actions): add homebrew release step

### PR DESCRIPTION
**Pull request by theRealPadster**
Adds action to bump the homebrew tap version on release

### CODE REVIEW

**Comment by theRealPadster** [comment]
Stupid conventional commits still hate me even when I add them. smh.

**Comment by itsmeow** [comment]
Try running it on a fork with a different target repo

**Comment by theRealPadster** [comment]
> Try running it on a fork with a different target repo

Tested on a fork of the [cli](https://github.com/theRealPadster/spicetify-cli) repo and the [tap](https://github.com/theRealPadster/spicetify-homebrew-tap) repo. It looks like we need to add an access token to the cli repo if we don't have one (I can't see them), since the default `GITHUB_TOKEN` doesn't work between repos. 

[This](https://github.com/theRealPadster/spicetify-homebrew-tap/commit/287fb66b6db239175f2f72970eef6ce56604f2c9) is the commit it made. Looks like it generates the formula from scratch, based on the action. For some reason the indentation for the install step is messed up though. Maybe from the workflow?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -88,12 +88,29 @@ jobs:
...
+        run: |
+          printf "::set-output name=%s::%s\n" tag-name "${GITHUB_REF#refs/tags/}"
+      - name: Update Homebrew tap
+        uses: mislav/bump-homebrew-formula-action@v2
+        if: "!contains(github.ref, '-')" # skip prereleases
+        with:
+          formula-name: spicetify-cli
+          formula-path: spicetify-cli.rb
+          homebrew-tap: spicetify/homebrew-tap
+          base-branch: master
```
Comment (ririxidev) [inline .github/workflows/build.yml:111]:
```suggestion
          base-branch: main
```

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Review by ririxidev — APPROVED** [review]

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -88,12 +88,29 @@ jobs:
...
     steps:
       - name: Update AUR package
         uses: fjogeleit/http-request-action@master
         with:
           url: http://vps.itsmeow.dev:8079/
           method: GET
+      - name: Extract version
+        id: extract-version
+        run: |
+          printf "::set-output name=%s::%s\n" tag-name "${GITHUB_REF#refs/tags/}"
```
Comment (itsmeow) [inline .github/workflows/build.yml:103]: this will break soon (march 2023). It needs to use output environment files

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Review by kyrie25 — APPROVED** [review]

**Review by itsmeow — APPROVED** [review]

**Pull request merged**